pkg/plugin/driver/utils/utils.go

```diff
@@ -30,7 +30,10 @@ import (
 )
 
 const (
-	terminatedContainerWindow = 1 * time.Minute
+	// terminatedContainerWindow is the amount of time after a plugins main container terminates
+	// that we consider it a failure mode. This handles the situation where the plugin container
+	// exits without returning results.
+	terminatedContainerWindow = 5 * time.Minute
```

Comment:
Increased 500%; agreed with @zubron 's comment offline that we should try and base this on some evidence/calculation rather than just my wild guesses. But FWIW I've successfully hand results reported with an AWS cluster even with the 1m window so I assume 5m is OK.